package/samplers/llambo/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2025 Jinglue Xu
```

Comment:
[@tennison](https://github.com/tennisonliu)
Would you like to be one of the authors as this method is from your paper?